canary_config.example.py

```diff
@@ -4,9 +4,12 @@
 INPUT_PIN = "16"
 OUTPUT_PIN = "15"
 
-# Uncomment these if you can't reuse the same two pins for all functionality.
-# PWM_PIN = "17"
-# INTERRUPT_PIN = "18"
+SW_PWM_PIN = "20"
+SW_INTERRUPT_PIN = "21"
+
+# Uncomment these if you can't reuse the same two pins for hw pwm and gpio functionality.
```

Comment:
Clarify that these are for when you can't use INPUT_PIN and OUTPUT_PIN for both. and maybe move lines 7 and 8 below this paragraph, so that's clearer (at the moment it looks like the pins you uncomment are replacements for lines 7 and 8).

Perhaps INPUT_PIN and OUTPUT_PIN should be renamed to make this less confusing? Like, maybe GPIO_INPUT_PIN and GPIO_OUTPUT_PIN. ...but if they get renamed, we need to go to each board and rename the configs, which would be a pain. Maybe don't do this now, but consider it for the future.

Reply:
yeah I agree the names are a bit confusing but would be annoying to change so will leave as is for now.